static/js/components/PostDisplay.js

```diff
@@ -46,19 +44,23 @@ export default class PostDisplay extends React.Component {
     return (
       <div className="post-summary">
         <div className="upvotes">
-          <button> &uArr;</button>
+          <img className="upvote-arrow" src="/static/images/upvote_arrow.png" />
```

Comment:
Is there a reason for the move away from a `<button>`? This has accessibility issues and has no native `disabled` behavior (which is something I'm doing in #93, which will merge soon too).